### PR TITLE
Remove inline scripts from some layouts

### PR DESCRIPTION
**Pull request by C-Lodder**
Redoing this: https://github.com/joomla/joomla-cms/pull/19641

Also converts the jQuery to ES5, ready to be moved to ES6.

### CODE REVIEW

File: media/system/js/fields/batch-language.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+/**
+ * @copyright  Copyright (C) 2005 - 2017 Open Source Matters, Inc. All rights reserved.
+ * @license    GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+(function() {
```
Comment (astridx) [inline media/system/js/fields/batch-language.js:6]: Shouldn't we use

(() => {
...
})()

Forget it. I just saw, that es6 is not used. You just removed the inline script for now.

File: media/system/js/fields/submenu.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+/**
+ * @copyright  Copyright (C) 2005 - 2017 Open Source Matters, Inc. All rights reserved.
+ * @license    GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+(function() {
```
Comment (astridx) [inline media/system/js/fields/submenu.js]: Same here? Shoudn't we use

(() => {
...
})()

File: media/system/js/fields/submenu.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+	document.addEventListener('DOMContentLoaded', function() {
+
+		if (window.toggleSidebar) {
+			toggleSidebar(true);
```
Comment (dneukirchen) [inline media/system/js/fields/submenu.js]: I found 0 occurrences of this in project, so we can probably remove it...

File: media/system/js/fields/submenu.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+		} else {
+			var sidebarHeader = document.getElementById('j-toggle-sidebar-header');
+			if (sidebarHeader) {
+				sidebarHeader.style.display = 'none';
+			}
+
+			var sidebarWrapper = document.getElementById('j-toggle-sidebar-wrapper');
+			if (sidebarWrapper) {
+				sidebarWrapper.style.display = 'none';
+			}
```
Comment (dneukirchen) [inline media/system/js/fields/submenu.js]: This will hide the sidebar by default. Is that on purpose?

**Comment by dneukirchen** [comment]
Not sure if `media/system/js/fields/submenu.js` is the correct place to put the script in. Is it a field? I would probably put it in `media/system/js`folder

File: media/system/js/fields/submenu.js (hunk 3 above)
Comment (C-Lodder) [reply media/system/js/fields/submenu.js]: This is the 2nd sidebar menu. We probably wont use it in core, but for BC purposes, I'd like to keep it.

**Comment by C-Lodder** [comment]
@dneukirchen I don't mind if the locations need changing. I just put it there cause the submenu is technically a form field.

File: media/system/js/fields/submenu.js (hunk 3 above)
Comment (dgrammatiko) [reply media/system/js/fields/submenu.js]: Please then put it in legacy folder

Comment (dgrammatiko) [reply media/system/js/fields/submenu.js]: No, just delete it

**Review by dneukirchen — CHANGES_REQUESTED** [review]
Please remove the sidebar toggle

**Review by dneukirchen — APPROVED** [review]

**Pull request merged**